packages/connect/src/SmoldotProvider/SmoldotProvider.test.ts

```diff
@@ -168,9 +168,9 @@ test('throws when got error JSON response', async () => {
 
   await provider.connect();
 
-   await expect(() => provider.send('hello', [ 'world' ]))
-    .rejects
-    .toThrow('666: boom!');
+  await expect(() => provider.send('hello', [ 'world' ]))
```

Comment:
This should be removed - right?

Reply:
The only changes in this file are whitespace fixes.  I havent touched the functionality.  That said .. this test still looks valid to me: We mock an error response and expect a rejected promise - that's an inteded behaviour that we want to test for.